src/connection.ts

```diff
@@ -1063,6 +1066,7 @@ class Connection extends EventEmitter {
 
     this.fedAuthRequired = false;
     this.fedAuthInfoToken = undefined;
+    this.collation = undefined;
```

Comment:
Would it be worth naming more narrowly as databaseCollation? I wonder if we’ll wind up needing to snag serverCollation manually at some point in order to work with tempdb parameters that show up in executed sql while working with a database whose collation is not the server collation.

Reply:
Sounds good to me! This property is intended to be "private" for now.

Regarding the tempdb, I don't think that's on `tedious` really? I think if your application creates tables in the tempdb, it should make sure those use the same collation as your database - that will ensure that `varchar` data send to tempdb tables will have the correct encoding.

Even if we would track the collation of the server (and thus the collation used for tempdb), SQL Server would still expect `varchar` parameters as encoded with the current database's encoding.